devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

```diff
@@ -139,6 +139,8 @@ def zwaveEvent(physicalgraph.zwave.Command cmd) {
 def refresh() {
 	log.debug "refresh()..."
 	delayBetween([
+			encap(zwave.associationV2.associationRemove(groupingIdentifier: 1, nodeId:[])), // Refresh Node ID in Group 1
+   			encap(zwave.associationV2.associationSet(groupingIdentifier: 1, nodeId:zwaveHubNodeId)), //Assign Node ID of SmartThings to Group 1
```

Comment:
I am kind of curious as to reasons that prompted this change? The hub automatically does this upon join for the past 4-5 years that I can remember. Anything that you'd like the hub team to look into?

Reply:
I'm also curious about this.

Reply:
my guess is that it's because it's not explicitly shown anywhere that we do this

Reply:
I assume that Association Group 1 is set to Node ID 01 automatically, but i have seen and gotten many reported cases where Group 1 is not successfully associated to Node ID 01 where the HEM Gen5 is not reporting automatically.

I have had to help large amount of supports where adjusting Group 01 to report to NodeID 01 using Z-Wave Tweaker custom device handler has allowed the HEM Gen5 to report normally,  which adding these 2 lines of code ensures that the HEM Gen5 is properly assigned.

-

I have noticed that Z-Wave controllers (not just SmartThings hub) that are using MULTICHANNEL ASSOCIATION to assign Group 1 to Node ID 1.1 which causes this issue as Node ID 1.1 is not responded to by HEM Gen5. For this to work, Node ID 1.0 for the use of this command class to work.

This code line is more of a bandage assuming that the association settings are automated.

Reply:
I don't think having this bandage in the DTH hurts anything, but I verified that the SmartThings hub adds node ID 1 / endpoint ID 0 to group 1 (lifeline) if the device supports Multichannel Association V3. So it should work. I don't have a device to verify with, but I'd be more than willing to investigate if you have customers running into this on our hub.

Reply:
@cbaumler - It should be working based on that information. If i do notice more of these in the future, i'll start debugging the backend to see if there are any problems via ZSniffer as well during its pair process.